test/io/configs/expected/no-defaults-with-db-other-authenticator.config

```diff
@@ -30,6 +30,7 @@ server-cors-allowed-origins = "http://example.com"
 server-host = "0.0.0.0"
 server-port = 80
 server-trace-header = "traceparent"
+server-timing-enabled = false
```

Comment:
These `test/io/configs/expected/no-defaults..` ones should give true (no default). For these we use settings in `test/io/db_config.sql`.